object_detection/.gitignore

```diff
@@ -1 +1,2 @@
-venv/
\ No newline at end of file
+venv/
+*.DS_Store
```

Comment:
Would it be better to move this to the main .gitignore and adding tho them /object_detection? 
Like:  /object_detection/venv/,  /object_detection/*.DS_Store

Reply:
Agree, I moved it to the main .gitignore